specification/webdriver-classic.bs

```diff
@@ -0,0 +1,177 @@
+<pre class="metadata">
+Title: webdriver-classic
+Shortname: wecg-webdriver-classic
+Level: 1
+Group: wecg
+Status: w3c/CG-DRAFT
+URL: https://w3c.github.io/webextensions/specification/webdriver-classic.html
+Editor: Kiara Rose, Apple, kiara_rose@apple.com
+Abstract: This specification defines how web extensions should be loaded and unloaded in WebDriver Classic.
```

Comment:
Can we add a prominent note that this is non-normative and that not all browser vendors are expected to implement this?